src/lib/analytics/utils.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import { isNumber, isString, remove, values as _values } from 'lodash'
+import { ANALYTICS_EVENT } from './constants'
+
+export const convertToGoogleAnalytics = (event: string, data: any = {}) => {
```

Comment:
i just confused a bit with that:

1. why event**Values** is "_everything except numbers_" and eventStrings is "_everything except strings_"
shouldn't it be vice versa 

2. shouldn't we store  eventValues.shift() to some var because it could be used again (for calculate eventLabel) ? Because .shift() mutates an array by removing the first value returned ?

Reply:
1. remove returns the removed items. 
2. this is a simple helper function with a specific purpose. why would it be used to calculate eventLabel? if someone will ever want such a logic he should take care of it in the future or make sure this helper method is already passed the correct label

Reply:
@sirpy as i've understand, this helper is used to convert data we generally sending to the Amplitude and/or Sentry to a Google Analytics format. So there no label arguments in the functions using it (fireEvent or error handler) - just some event name and data object are passed

Reply:
i dont understand your point.
this function does what it does. we dont need to make it compatible to any possible future scenario some developer might want to do

Reply:
@sirpy So, what you want to propose here ? To make GA wrapper and fireGoogleAnalyticsEvent accepting (event, eventAction, eventValue, eventLabel) and just send them as is ? And to perform that conversion inside fireEvent() (only if GA enabled) and call  fireGoogleAnalyticsEvent with event/data converted to event/action/value/label ?